includes/templates/admin/edit-story.php

```diff
@@ -76,6 +76,12 @@
 
 ?>
 
+<div id="stories-editor-no-js" class="wrap">
```

Comment:
Why not use the pre-existing `no-js` class here?

Reply:
This is just an id, so I can hide and show. `no-js` is added removed to the html / body tags.

Reply:
Sorry, what I meant is `hide-if-js`. WordPress has this class, let's use it to make lives easier.

There's also `hide-if-no-js`

Reply:
Updated.